### PR TITLE
De54331 cli fips error

### PR DESCRIPTION
**Pull request by jnpblrdrgz**

### CODE REVIEW

**Review by lordkyzr — APPROVED** [review]

**Comment by lordkyzr** [comment]
Remember to squash this when merging please

File: tests/test_jose_auth_transport.py [hunk 1]
```diff
@@ -192,6 +192,26 @@ def test_supported_jwe_encryptions_success(self):
...
+            self._encrypt_decrypt()
+
+    def test_no_kid_in_headers_uses_all_keys_to_decrypt(self):
+        self._jwenc_patch = patch("launchkey.transports.jose_auth.JWEnc",
+                                return_value=MagicMock(spec=JWT)).start()
+        self._jwenc_patch.return_value.unpack.return_value.headers = \
+            {"alg": "RS512",
+             "typ": "JWT",
+             }
+        self._encrypt_decrypt()
```
Comment (bporter125) [inline tests/test_jose_auth_transport.py:213]: Is this asserting anything?

Comment (lordkyzr) [reply tests/test_jose_auth_transport.py:213]: Updated to assert more specifically what its testing

File: features/steps/key_steps.py [hunk 2]
```diff
@@ -64,7 +64,8 @@ def attempt_basic_api_call(context):
 
 @then("no valid key will be available to decrypt response")
 def no_key_exists_to_decrypt_response(context):
-    assert_that(str(context.current_exception), equal_to("Incorrect decryption."),
-                "Expected an Incorrect Decryption error but received a different error.")
 
-    context.execute_steps("Then a ValueError error occurs")
+    assert_that(str(context.current_exception),
+                string_contains_in_order("The key id:", "could not be found in the entities available keys."))
```
Comment (CMoncur) [inline features/steps/key_steps.py:69]: Nice, this is a sweet function 👍

**Review by CMoncur — APPROVED** [review]
:shipit:

**Pull request merged**